cmd/utils.go

```diff
@@ -1883,13 +1883,11 @@ func CheckStackRequirements(log *LoggingConfig, requirementArray map[string]stri
 	log.Info.log("Checking stack requirements...")
 
 	for technology, minVersion := range requirementArray {
-		if minVersion == "" {
-			log.Info.log("Skipping ", technology, " - No requirements set.")
-		} else if technology == "Docker" && buildah {
+		if technology == "Docker" && buildah {
 			log.Info.log("Skipping Docker requirement - Buildah is being used.")
```

Comment:
Could we be using Debug.log instead of Info so that we don't clutter up the logs?

Reply:
Yeah sure - I did think about this; now Info.log is only used to let the user know that the stack requirements are being checked, each technology check is now a Debug.log.